lib/proxy.js

```diff
@@ -137,12 +137,13 @@ function buildResultingMiddleware (host, options, stream) {
     var reqOpts = {
       url     :  URL.format(proxyUrl),
       method  : req.method,
-      headers : _.defaults(options.headers || {}, req.headers),
+      // headers : _.defaults(options.headers || {}, req.headers),
+      headers : options.headers,
```

Comment:
Like i said this is a temp solution. i am just showing that this is the reason that super test was not working.

I am not sure what solution we want to go with because i am assuming that super test wont be the only thing that could cause issues.

Since this is a proxy library and essentially we will be relying on the request coming straight from a browser i don't know if it would be safe to assume all headers will be accepted by an API endpoint.